src/e2e-test/java/io/cdap/plugin/bqexecute/actions/BQExecuteActions.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```

Comment:
Change to 2022

Same comment applies to all licences.

Reply:
@sawantpritam -updated